grammar.y

```diff
@@ -1129,10 +1131,16 @@ table_constraint_list_opt:
 table_constraint_list:
   ',' table_constraint
   {
+    if _, ok := $2.(*TableConstraintPrimaryKey); ok {
+      createStmtHasPrimaryKey = true
+    }
     $$ = []TableConstraint{$2}
   }
 | table_constraint_list ','  table_constraint
   {
+    if _, ok := $3.(*TableConstraintPrimaryKey); ok && createStmtHasPrimaryKey {
```

Comment:
If we have already seen a `TableConstraintPrimaryKey` and see another one we set an error.

Reply:
Reminder: this is not a syntax error.